src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -89,7 +89,7 @@ public async Task TestStringMultiFiltersWithAndCondition()
                         and: [
                             { additionalAttributes: {name: {eq: ""volcano1""}}}
                             { moons: {name: {eq: ""1 moon""}}}
-                            { moons: {details: {contains: ""v1""}}}
+                            { moons: {details: {contains: ""11""}}}
```

Comment:
why this change?

Reply:
This query was returning blank results so changed this condition to get proper results. This test was added by me in my last PR.